routes/web.php

```diff
@@ -3,7 +3,7 @@
 
 Route::redirect('/', '/en');
 
-Route::group(['prefix' => '{locale}'], function () {
+Route::group(['prefix' => app()->getLocale() ?? 'en'], function () {
```

Comment:
Whoa, this is a *really* interesting solution. I had never even considered this. I'll give it a look a bit later--thank you so much!